Show last saved date

oicramps opened the pull request:
# Summary

- Adds a status bar to `logic` and `conceptual` components that show the date when the model was last saved.
- Creates a component to the `bug report button` that is shown on almost every page. Suggested by @idmarjr [here](https://github.com/brmodeloweb/brmodelo-app/pull/263#discussion_r753545244)

# Screenshots

~~https://user-images.githubusercontent.com/7519115/142931030-50724ddc-1d51-4af4-830c-c4ca5d9b7fa7.png~~

![save-info-1](https://user-images.githubusercontent.com/301545/144685419-aaefa9c5-7ae5-426f-8cbd-716d47af072d.png)

oicramps committed:
feat: show a confirmation when changing route

oicramps committed:
feat: creates a factory to handle preventExit events

oicramps committed:
feat: prevent exit application handling in the logic model

oicramps committed:
fix: revert remove imports

oicramps committed:
remove unused code

oicramps committed:
feat: adds vscode folder to gitignore

oicramps committed:
feat: creates a bug report button component

oicramps committed:
feat: creates a status bar component

oicramps committed:
feat: replace repeated code with the new bug-report-button component

oicramps committed:
feat: adds a status bar to the logic model

oicramps committed:
Merge branch 'development' into feature/last-save-date

oicramps committed:
refactor: remove unused controller content

oicramps committed:
fix: rever code format

oicramps committed:
fix: rever code format

oicramps committed:
fix: rever code format

oicramps committed:
Merge branch 'feature/last-save-date' of github.com:brmodeloweb/brmodelo-app into feature/last-save-date

miltonbsn commented on app/angular/conceptual/conceptual.js:
```diff
@@ -385,16 +386,6 @@ const controller = function (ModelAPI, $stateParams, $rootScope, $timeout, $uibM
-
-		configs.paper.on('link:mouseleave', (linkView) => {
-			linkView.removeTools();
-		});
```
I think there is problematic merge around here

oicramps replied:
I'll check, @miltonbsn. Thank you

miltonbsn reviewed (changes requested):
Issues I would like to discuss:

- visual place for the last saved date
- app/angular/service/logicService.js was entirely formatted (with prettier I think)
- there were a merge mistake, this pr is removing the issues #229 and pr #259

oicramps committed:
fix: revert LogicService

oicramps committed:
fix: revert conceptual

oicramps committed:
fix: adds bugReportButton and statusBar module dependencies

miltonbsn commented on app/angular/logic/logic.js:
```diff
@@ -267,10 +271,6 @@ const controller = function (
 
-	ctrl.zoomNone = function () {
-		LogicService.zoomNone();
-	}
```
I believe it is another effect of a merge

oicramps committed:
Fix merge

oicramps committed:
fix: fix logic model saving not setting updatedAt date

idmarjr committed:
Merge branch 'development' into feature/last-save-date

Solved conflicts:
	app/angular/conceptual/conceptual.html
	app/angular/logic/logic.html
	app/angular/logic/logic.js
	app/angular/login/login.html
	app/angular/recovery/recovery.html
	app/angular/recovery/reset.html
	app/angular/signup/signup.html
	app/angular/workspace/workspace.html

idmarjr committed:
Merge branch 'development' into feature/last-save-date

idmarjr committed:
Add translations to be used at status-bar

idmarjr committed:
.status-bar styles and placement

- Move .status-bar from footer to workspace header, right bellow model name;
- Simplify .status-bar markup and CSS;
- Add .document-info <div> to hold document name and last save info;

idmarjr committed:
Add .status-bar to conceptual model workspace

idmarjr committed:
Fix aligment of .model-properties

After introduce the last save date at workspace header a
layout misalignment took place at .model-properties.

This commit fix this issue

idmarjr committed:
Merge branch 'development' into feature/last-save-date

oicramps committed:
fix: show last updated date instead of current date

idmarjr committed:
Merge branch 'development' into feature/last-save-date

Fixed conflicts:
	app/i18n/languages/en.js
	app/i18n/languages/pt_BR.js
